users-guide.md

```diff
@@ -477,8 +477,8 @@ limitations under the License.
 [Annotation Processor]: http://docs.oracle.com/javase/6/docs/api/javax/annotation/processing/package-summary.html
 [Builder Pattern]: http://en.wikipedia.org/wiki/Builder_pattern
 [CoffeeMaker Example]: https://github.com/google/dagger/tree/master/examples/simple/src/main/java/coffee
-[Component]: /api/latest/dagger/Component.html
-[Component#provision-methods]: </api/latest/dagger/Component.html#provision-methods>
+[Component]: api/latest/dagger/Component.html
+[Component#provision-methods]: <api/latest/dagger/Component.html#provision-methods>
```

Comment:
Hm, I thought we had fixed these. I guess not. Thanks for getting these.

While you're at it, do you mind removing the angle brackets? I don't believe those are necessary.